app/means_test/validators.py

```diff
@@ -25,10 +25,25 @@ def __init__(
     def __call__(self, form, field):
         other_field = form._fields.get(self.dependent_field_name)
         if other_field is None:
-            raise ValueError('no field named "%s" in form' % self.dependent_field_name)
+            raise ValueError(f'No field named "{self.dependent_field_name}" in form.')
+
+        # If the dependent field is a SelectMultipleField, its data will be a list
+        dependent_data = other_field.data
+
+        # Update the condition check logic to handle lists
+        if isinstance(dependent_data, list):
+            # If using `IN` condition, check if dependent_value is in the list
+            if self.condition_type == ValidateIfType.IN:
+                match = self.dependent_field_value in dependent_data
+            # If using `EQ` condition, check if there's a match
+            elif self.condition_type == ValidateIfType.EQ:
+                match = self.dependent_field_value in dependent_data
```

Comment:
Shouldn't this check if they are equal rather than if one list is a subset of another list?